src/Contracts/Database/Connection.php

```diff
@@ -49,7 +49,8 @@ public function selectOne($query, $bindings = []);
      *
      * @param  string  $query
      * @param  array   $bindings
-     * @return object
+     *
```

Comment:
The connection class uses an extended Gibbon\Database\Result class, rather than the base PDOStatement class. It internally handles checking for false, and returns an empty Result object on failure, to avoid areas in the code constantly having to check for false values. So there shouldn't be cases where these methods are returning false 🤔

Reply:
Thanks for pointing this out. Updated the phpdocs. Please check.